home-work-app/src/app/components/styles/filterPanel.style.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../configs/style.config.scss';
```

Comment:
Additional scss files is unnecessary with `styled-components`

Reply:
Will be removed